PSKoans/koanOrder.md

```diff
@@ -0,0 +1,48 @@
+# PowerShell Koan Order
+See below for a table of all koans and their corresponding positions. When working on creating new Koans try to update these table to help other devs keep track.
+
+| Folder                    |  Koan                           |  Int Position  |  Module      |
+|---------------------------|---------------------------------|----------------|--------------|
+|  Foundations              |[AboutAssertions](Koans/Foundations/AboutAssertions.Koans.ps1)|101| |
+|  IntroTraining            |[AboutOOP](Koans/IntroTraining/AboutOOP.Koans.ps1)|104| |
```

Comment:
I think this'll probably be the last thing. Can you rename the `IntroTraining` folder to `Introduction` and then move the `AboutAssertions` and `AboutStrings` topic into there as well?

Thanks! 😊

Reply:
🦄